constants/ram_constants.asm

```diff
@@ -118,21 +125,23 @@ DEF INIT_OTHER_ITEM_LIST EQU 3
 DEF INIT_PLAYEROT_LIST   EQU 4
 DEF INIT_MON_LIST        EQU 5
 
+; wEnteredMapFromContinue::
+DEF ENTERED_MAP_FROM_CONTINUE_2 EQU 2
+DEF ENTERED_MAP_FROM_CONTINUE_7 EQU 7
```

Comment:
I'm pretty sure these are unused.

Reply:
I think so as well; but I can double check in game with a few tests to see if I can catch something might have missed.

Reply:
If so, `wEnteredMapFromContinue` should be renamed, perhaps to `wUnusedMapEntryFlags`.